tests/test_bot.py

```diff
@@ -145,6 +145,12 @@ def test_get_me_and_properties(self, bot):
         assert f'https://t.me/{get_me_bot.username}' == bot.link
         assert commands == bot.commands
 
+    def test_comparison(self, bot):
```

Comment:
I think this test should be elaborated. please have a look at the other TG object classes tests for inspiration ;)